README.md

```diff
@@ -1,237 +1,66 @@
-# Sansible Scaffold
+# amazon_inspector_agent
 
-[![Build Status](https://travis-ci.org/sansible/sansible.svg?branch=scaffold)](https://travis-ci.org/sansible/sansible)  
+Master: [![Build Status](https://travis-ci.org/sansible/amazon_inspector_agent.svg?branch=master)](https://travis-ci.org/sansible/amazon_inspector_agent)
+Develop: [![Build Status](https://travis-ci.org/sansible/amazon_inspector_agent.svg?branch=develop)](https://travis-ci.org/sansible/amazon_inspector_agent)
 
-* [TL;DR](#tldr)
-* [Rationale](#rationale)
-* [Requirements](#requirements)
-* [Creating Your Role](#creating-your-role)
-* [Testing Your Role](#testing-your-role)
-* [pre-commit Integration](#pre-commit-integration)
-* [Travis CI Integration](#travis-ci-integration)
+* [ansible.cfg](#ansible-cfg)
+* [Installation and Dependencies](#installation-and-dependencies)
+* [Tags](#tags)
+* [Examples](#examples)
 
+This role downloads and installs[Amazon Inspector Agent](https://docs.aws.amazon.com/inspector/latest/userguide/inspector_agents.html)
+on supported GNU/Linux systems.  No parameters are required, but the following are supported:
 
-## Rationale
+| parameter                                     | default                                                    | comments                                                           |
+|-----------------------------------------------|------------------------------------------------------------|--------------------------------------------------------------------|
```

Comment:
You shouldn't need all the extra spaces here, Markdown will handle this for you if you use headers with three hyphens:

|parameter| default|comments|
|---|---|---|
|sansible_amazon_inspector_agent_url|https://d1wk0tztpsntt1.cloudfront.net/linux/latest/install | URL from which to download the installation script|
|sansible_amazon_inspector_agent_dest|/tmp/amazon_inspector_agent| Path for the installation script|
|sansible_amazon_inspector_agent_enabled|yes|Whether the agent should be started automatically at boot (yes/no)|
|sansible_amazon_inspector_agent_state|started|The state of the agent after installation (started/stopped)|
|sansible_amazon_inspector_agent_download_only|no|Download only, skip installation and configuration (yes/no)|

```
# RAW code
|parameter| default|comments|
|---|---|---|
|sansible_amazon_inspector_agent_url|https://d1wk0tztpsntt1.cloudfront.net/linux/latest/install | URL from which to download the installation script|
|sansible_amazon_inspector_agent_dest|/tmp/amazon_inspector_agent| Path for the installation script|
|sansible_amazon_inspector_agent_enabled|yes|Whether the agent should be started automatically at boot (yes/no)|
|sansible_amazon_inspector_agent_state|started|The state of the agent after installation (started/stopped)|
|sansible_amazon_inspector_agent_download_only|no|Download only, skip installation and configuration (yes/no)|
```

Reply:
I know I don't have to, but it looks so much nicer in a text editor... do you have strong feelings about those extra hyphens and spaces?

Reply:
I've maintained tables like that before, it's a total pain the bum longterm; but I am fine with it if you want to keep it that way.